lib/config.cpp

```diff
@@ -554,13 +582,38 @@ void Config::loader(const json_uri& uri, json& schema) {
     EVTHROW(EverestInternalError(fmt::format("{} is not supported for schema loading at the moment\n", uri.url())));
 }
 
+void Config::ref_loader(const json_uri& uri, json& schema) {
+    BOOST_LOG_FUNCTION();
+
+    if (uri.location() == "http://json-schema.org/draft-07/schema") {
+        schema = nlohmann::json_schema::draft7_schema_builtin;
+        return;
+    } else {
+        auto path = uri.path();
+        if (this->types.contains(path)) {
+            schema = this->types[path];
+            EVLOG_verbose << fmt::format("ref path \"{}\" schema has been found.", path);
+            return;
```

Comment:
Is there any order necessary for loading the the type files? For example yeti types depend on unit types.  What if yeti.json gets loaded first?

Reply:
That should not be an issue here since all type json files have been loaded at startup into the this->types member